internal/cluster/distribution/eks/service.go

```diff
@@ -144,6 +146,9 @@ func NewNodePoolFromCFStack(name string, labels map[string]string, stack *cloudf
 	nodePool.SubnetID = nodePoolParameters.Subnets // Note: currently we ensure a single value at creation.
 	nodePool.Status = NewNodePoolStatusFromCFStackStatus(aws.StringValue(stack.StackStatus))
 	nodePool.StatusMessage = aws.StringValue(stack.StackStatusReason)
+	if nodePoolParameters.CustomNodeSecurityGroups != "" {
+		nodePool.SecurityGroups = strings.Split(nodePoolParameters.CustomNodeSecurityGroups, ",")
+	}
```

Comment:
**Note**: we are going to need this change for #3334 , but I think it should be a separate PR, it's going to affect the API as well, regenerated code, etc.

Reply:
Implemented it [here](https://github.com/banzaicloud/pipeline/pull/3337).

Reply:
fixed